docker-compose.yml

```diff
@@ -83,3 +83,8 @@ services:
     image: tile38/tile38:latest
     ports:
       - "19851:9851"
+
+  redis-stack:
+    image: redis/redis-stack-server:7.4.0-v0
```

Comment:
Just in case, `7.4.0-v0` is current version of `redis-stack-server` and it includes redis 7.4 + modules with recent versions.